smallrye-reactive-messaging-http/src/main/java/io/smallrye/reactive/messaging/http/converters/AssignationUtils.java

```diff
@@ -108,6 +102,13 @@ private static boolean isAssignable(Class<?> cls, final Class<?> toClass, final
         return toClass.isAssignableFrom(cls);
     }
 
+    private static boolean isInteger(Class<?> toClass) {
```

Comment:
Can this class be replaced with `ClassUtils` from provider artifact?

This check definitely exists in both, maybe more methods are duplicated too?